CHANGELOG.md

```diff
@@ -4,22 +4,68 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## [Unreleased]
+## [0.8.0-rc1] - 2019-12-11: "Blockchain Good, Orange Coin Bad"
+
+This release named by @m-schmoock.
```

Comment:
@cdecker  If you want, you can note my full name: `Michael Schmoock @m-schmoock`

Also, though being used throughout the changelog, `This release named by XYZ` is not a correct sentence.
Suggestion: "This release **was** named by XYZ"

Reply:
fixes that: https://github.com/ElementsProject/lightning/pull/3334